ConsoleStub/Console.vcxproj

```diff
@@ -169,6 +174,9 @@
       <GenerateMapFile>true</GenerateMapFile>
       <MapExports>true</MapExports>
     </Link>
+    <PreBuildEvent>

```

Comment:
Not needed for this project as the version is that of the deployed app, not of Dolphin